sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -657,3 +661,9 @@ def ParsePaths(paths):
             tokens.append(token)
 
     return tokens
+
+
+def validate_cache_staleness_value(max_integrated_cache_staleness):
+    int(max_integrated_cache_staleness)  # Will throw error if data type cant be converted to int
+    if max_integrated_cache_staleness <= 0:
+        raise ValueError("Parameter 'max_integrated_cache_staleness_in_ms' can only be a positive integer.")
```

Comment:
We usually let the service do these kind of type/value validations - what would the service do if I set this value to `-1`? Or a string?

Reply:
I just double checked and it seems like the backend does no validation on this value, and instead sets the default value of 5 mins so long as the dedicated gateway is configured for the account - one doesn't even have to pass the option in explicitly for the integrated cache to be used, since it seems it gets utilized regardless.